docs/doc_5_best_practices.md

```diff
@@ -58,11 +58,13 @@ When you have confirmed that the `time.Time pointer` is non-nil, you can safely
 
 In the Atlas Go SDK, the `io.ReadCloser` type is used to return binary data from APIs. 
```

Comment:
```suggestion
In the Atlas Go SDK, the `io.ReadCloser` type can be used to return binary data using the APIs. 
```

Reply:
@kanchana-mongodb this is not a "can be" the code is auto-generated it will always be `io.ReadCloser`